PerformanceNavigationTiming.notRestoredReasons: untangle/expand reasons for null (#44669)
mdn/content: files/en-us/web/api/performancenavigationtiming/notrestoredreasons/index.md

When the `PerformanceNavigationTiming` object does not represent a history navigation, `notRestoredReasons` will return `null`. This is useful for determining whether bfcache is not relevant to a particular navigation (as opposed to `notRestoredReasons` not being supported, in which case it would return `undefined`).

> [!NOTE]

> > `notRestoredReasons` may return `null` despite the navigation type being reported as a back/forward navigation. These circumstances include:

Reviewer: ```suggestion
It is worth noting that `notRestoredReasons` may return `null` even when the navigation type is reported as back/forward. These circumstances include:
```

And then put this content into normal text, not as a note (remove the "> [!NOTE]" line and the ">" characters from the line starts). My reasoning:

- I think this info is getting a bit long to work as a note. At this point, it is more than 50% of the "Value" section.
- I also think it is important, so it should be part of the central flow. Notes are often viewed as asides, and sometimes glossed over by readers.

When the `PerformanceNavigationTiming` object does not represent a history navigation, `notRestoredReasons` will return `null`. This is useful for determining whether bfcache is not relevant to a particular navigation (as opposed to `notRestoredReasons` not being supported, in which case it would return `undefined`).

> [!NOTE]

> > `notRestoredReasons` may return `null` despite the navigation type being reported as a back/forward navigation. These circumstances include

Author: ```suggestion
> `notRestoredReasons` may return `null` despite the navigation type being reported as a back/forward navigation. These circumstances include:
```
